### PR TITLE
Add Community libraries list page building to adabot

### PR DESCRIPTION
**Pull request by jposada202020**
Add a new community bundle libraries list. 
partially intend to solve https://github.com/adafruit/CircuitPython_Community_Bundle/issues/67

### CODE REVIEW

File: adabot/lib/common_funcs.py [hunk 1]
```diff
@@ -91,22 +91,31 @@ def parse_gitmodules(input_text):
...
     # Assume the bundle repository is public and get the .gitmodules file
     # without any authentication or Github API usage.  Also assumes the
     # master branch of the bundle is the canonical source of the bundle release.
-    result = requests.get('https://raw.githubusercontent.com/adafruit/Adafruit_CircuitPython_Bundle/master/.gitmodules',
-                          timeout=15)
-    if result.status_code != 200:
-        #output_handler("Failed to access bundle .gitmodules file from GitHub!", quiet=True)
-        raise RuntimeError('Failed to access bundle .gitmodules file from GitHub!')
-    return parse_gitmodules(result.text)
+    if bundle == "Adafruit_CircuitPython_Bundle":
```
Comment (tannewt) [inline adabot/lib/common_funcs.py:105]: How about making bundle `adafruit/Adafruit_CircuitPython_Bundle` and `adafruit/CircuitPython_Community_Bundle`? That way you can put it into the url and don't need an if statement. (It'll also work for other bundles then.)

File: adabot/lib/common_funcs.py [hunk 2]
```diff
@@ -91,22 +91,31 @@ def parse_gitmodules(input_text):
         results.append((submodule_name, submodule_variables))
     return results
 
-def get_bundle_submodules():
-    """Query Adafruit_CircuitPython_Bundle repository for all the submodules
-    (i.e. modules included inside) and return a list of the found submodules.
+def get_bundle_submodules(bundle):
+    """Query Adafruit_CircuitPython_Bundle or CircuitPython_Community_Bundle
+    repository for all the submodules (i.e. modules included inside) and
```
Comment (tannewt) [inline adabot/lib/common_funcs.py:96]:
```suggestion
    """Query bundle repository for all the submodules (i.e. modules included inside) and
```

**Review by tannewt — CHANGES_REQUESTED** [review]
Thanks for this! I think we should make it more generic now.

**Comment by jposada202020** [comment]
Thank you for reviewing and the comments, I work on the comments and changes during the weekend.

File: adabot/lib/common_funcs.py (hunk 1 above)
Comment (jposada202020) [reply adabot/lib/common_funcs.py]: Changes were made according to suggestions, see https://github.com/adafruit/adabot/pull/218/commits/57d59aebac6c3f1abc46b3bc5f9f8ce5da7832fc. Thanks for reviewing

File: adabot/lib/common_funcs.py (hunk 2 above)
Comment (jposada202020) [reply adabot/lib/common_funcs.py]: Changes were made according to suggestions, see https://github.com/adafruit/adabot/pull/218/commits/57d59aebac6c3f1abc46b3bc5f9f8ce5da7832fc. Thanks for reviewing

**Comment by jposada202020** [comment]
@tannewt  Thanks for reviewing. Similar to Neradoc PR, I would like some testing, I tested myself, and I could test, but it burns the limit very fast, maybe both PR could be tested at the same time thanks.

**Review by tannewt — CHANGES_REQUESTED** [review]
I don't think we should change the filename and instead have it be consistent in every repo. That way it'll be the same for all bundles and the adafruit bundle repo won't contain anything about the community bundle.

It'd also be good to change bundles to a dictionary where the key is the bundle name and the value is the title used in the library list.

**Comment by jposada202020** [comment]
@tannewt Gotcha. Will do :)

**Comment by jposada202020** [comment]
Closing for now, until discussion regarding the graphics libraries move . Will reopen a New PR when bundles direction is in place

**Pull request closed without merging**